readthedocs/core/views/hooks.py

```diff
@@ -190,7 +191,7 @@ def github_build(request):
 
 
 @csrf_exempt
-def gitlab_build(request):
+def gitlab_build(request):  # noqa: D205
```

Comment:
Without this `#noqa` comment, the pep257 linter gets unhappy about what it sees as miscapitalisation of "GitHub" and "GitLab".